packages/web/src/components/artist/ArtistCard.module.css

```diff
@@ -92,6 +92,7 @@
 }
 
 .artistHandle {
+  display: inline-block;
```

Comment:
What's the behavior when the handle is really long? Does the badge wrap to the next line?

Reply:
Went with option 1... margin-right on the handle so when it wraps it's all left aligned.